docs/topics/proc_selecting-the-jdk-builder-image-for-your-application.adoc

```diff
@@ -0,0 +1,57 @@
+[id='selecting-the-jdk-builder-image-for-your-application_{context}']
+= Selecting the JDK image for your {runtime} application
+Edit the `pom.xml` file of your application to specify the image for building and running your {runtime} application.
+
+.Prerequisites
+* A Maven-based application.
+* Maven installed.
+
+.Procedure
+. Open the `pom.xml` file in the application root directory and locate the `fabric8.generator.from` property.
+
+. Replace the `fabric8.generator.from` property to specify the version of RHEL and OpenJDK you intend to use.
+
+* RHEL 7 with OpenJDK 8
++
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<fabric8.generator.from>{name-2si-image-openjdk8-rhel7}</fabric8.generator.from>
+----
+
+* RHEL 7 with OpenJDK 11
++
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<fabric8.generator.from>{name-2si-image-openjdk11-rhel7}</fabric8.generator.from>
+----
+
+* RHEL 8 with OpenJDK 8
++
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<fabric8.generator.from>{name-2si-image-openjdk8-rhel8}</fabric8.generator.from>
+----
+
+* RHEL 8 with OpenJDK 11
++
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<fabric8.generator.from>{name-2si-image-openjdk11-rhel8}</fabric8.generator.from>
+----
+
+. Build the application using Maven from the root directory of the application.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+{parameter-maven-command}
```

Comment:
I am not sure if `mvn {runtime}:run` starts a fabric8-maven so the last two steps are useless.
Also, it seems as we want to run the app as a standalone app.
Please could you move this to the section :
> Preparing {runtime} application for OpenShift deployment
WDYT @Ladicek

Reply:
Yea, this is all about OpenShift, doesn't make sense outside. I agree it would be best to merge this with the related content in the "OpenShift deployment" chapter.